integration-tests/ast/hello_world.mj.stdout

```diff
@@ -0,0 +1,19 @@
+class HelloWorld {
+	public int bar(int a, int b) {
+		return c = (a + b);
+	}
+	public static void main(String[] args) {
+		(System.out).println(43110 + 0);
+		boolean b = true && (!false);
+		if ((23 + 19) == ((42 + 0) * 1))
+			b = (0 < 1);
+		else if (!(array[2 + 2])) {
+			int x = 0;
+			x = (x + 1);
+		} else {
+			(new HelloWorld()).bar(42 + (0 * 1), -1);
```

Comment:
how is this single-statement `else` block different from the single-statement `if` block above (line 8--10 in this file)

Reply:
So, is this wrong or correct? I am getting really confused now. Especially since the txt files on the compilerpraktikum website are obviously not correct too

Reply:
In the input file were braces, so the pretty printer will also print braces (`block` statement)